code/modules/surgery/organs/organ.dm

```diff
@@ -326,35 +311,23 @@ I use this so that this can be made better once the organ overhaul rolls out --
 		return 0
 	return src == O.get_int_organ(organ_tag)
 
-/obj/item/organ/proc/is_robotic(var/purist = FALSE)
-	if(purist && (robotic > 1 || status & (ORGAN_ROBOT))) //Only the robotiest.
-		return TRUE
-	if(robotic || status & (ORGAN_ROBOT|ORGAN_ASSISTED)) //Any tech will do.
+/obj/item/organ/proc/is_robotic()
+	if(status & ORGAN_ROBOT)
 		return TRUE
-
 	return FALSE
 
 /obj/item/organ/serialize()
 	var/data = ..()
 	if(status != 0)
 		data["status"] = status
-	if(robotic > 0)
-		data["robotic"] = robotic
 
 	// Save the DNA datum if: The owner doesn't exist, or the dna doesn't match
 	// the owner
 	if(!(owner && dna.unique_enzymes == owner.dna.unique_enzymes))
 		data["dna"] = dna.serialize()
 	return data
 
-/obj/item/organ/deserialize(var/data)
-	switch(data["robotic"])
-		if(1)
-			mechassist()
-		if(2)
-			robotize()
-		else
-			// Nothing
+/obj/item/organ/deserialize(data)
 	if(isnum(data["status"]))
```

Comment:
A bit uneasy about dropping the `robotize` call here, but looking at what that proc does, it doesn't look like it's anything that the serialization system doesn't cover already

Reply:
I thought about that, as well.

I could through in a `if(isrobotic())` check after it sets status, and just call `robotize()`, if you wanted.

That'd ensure they have the proper icon (the only thing really lost in serialization).

Reply:
Well, `robotize` has an `isrobotic()` guard, so that wouldn't quite work - you'd need to do the check inside of the status deserialization statement

Reply:
Or maybe add a force argument to override the check and make it robit the thing anyhow haha. How horrifying